apis/facebook/api/src/facebook_api.e

```diff
@@ -124,6 +124,50 @@ feature -- Facebook: Get User
 			end
 		end
 
+	user_likes (a_path: STRING; a_params: detachable FB_PAGE_PARAMETER): detachable STRING
+		do
+			if
+				attached a_params and then
+				attached a_params.parameters as l_parameters
+			then
+				api_get_call (facebook_url (a_path, l_parameters ), Void)
+			else
+				if a_path.starts_with ("https") then
+					api_get_call (a_path, Void)
+				else
+					api_get_call (facebook_url (a_path, Void ), Void)
+				end
+			end
+			if
+				attached last_response as l_response and then
+				attached l_response.body as l_body
+			then
+				Result := l_body
+			end
+		end
+
+	user_groups (a_path: STRING; a_params: detachable FB_GROUP_PARAMETER): detachable STRING
+		do
+			if
+				attached a_params and then
+				attached a_params.parameters as l_parameters
+			then
+				api_get_call (facebook_url (a_path, l_parameters ), Void)
+			else
+				if a_path.starts_with ("https") then
+					api_get_call (a_path, Void)
+				else
+					api_get_call (facebook_url (a_path, Void ), Void)
+				end
+			end
+			if
+				attached last_response as l_response and then
+				attached l_response.body as l_body
+			then
+				Result := l_body
+			end
+		end
+
 feature -- Feeds: publish, delete, update
 
 	user_feed_publish (a_user_id: STRING; a_params: detachable FB_USER_FEED_PUBLISHING): detachable STRING
```

Comment:
if this is a procedure ...
maybe `publish_on_user_feed` to follow same convention as `publish_photo_on_user_feed`
Also, missing comment to explain the arguments and what is the result.